src/indexer.cc

```diff
@@ -830,6 +831,125 @@ class IndexDataConsumer : public index::IndexDataConsumer {
            D->getKind() == Decl::CXXConversion))
         role = Role(role | Role::Implicit);
       do_def_decl(func);
+
+      if (is_def) {
+        class FuncVisitor : public RecursiveASTVisitor<FuncVisitor> {
+          public:
+            FuncVisitor(IndexDataConsumer* parent, 
+    IndexFile *db, SourceManager& SM, const LangOptions &Lang, int lid, const Decl* D) : parent(parent), db(db), SM(SM), Lang(Lang), lid(lid), root(D) {} 
+            bool VisitFunctionDecl(FunctionDecl* decl) {
```

Comment:
I've changed `.clang-format` to use the LLVM style.. so the argument should be written as `FunctionDecl *decl`. It may look weird as they have different CamelCase/camelCase conventions. I also find it a bit unnatural to fix existing Chromium style into LLVM but what I chose is that: for llvm related variables/types, use its naming convention (`VariableName` `funcName`) but for others use the Chromium one.